CMakeLists.txt

```diff
@@ -52,7 +52,7 @@ endif()
 
 # FFIG won't work without Python and a C++ compiler so we can assume
 # support for Python and C++ must exist.
-set(all_ffig_bindings "PYTHON;CPP;CPP_MOCKS")
+set(all_ffig_bindings "PYTHON;CPP;CPP_MOCKS;BOOST_PYTHON")
```

Comment:
We require Python and C++, but we don't absolutely require boost... I think this should be an optional binding.

Reply:
Agree. Feel free to patch the PR.

Reply:
Until we’ve completed #412 I’ll keep boost Python as required until we get CI passing.